netIDS/TimeIntervalsIDS.py

```diff
@@ -0,0 +1,168 @@
+
+#!/usr/bin/python3
+
+# Copyright (c) 2018 SecurityMatters
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors: Alexios Lekidis (alexios.lekidis@secmatters.com)
+#
+
+import sys, getopt
+from optparse import OptionParser
+import time
+from pathlib import Path
+import sqlite3
+from rawSniffer import sniff
```

Comment:
I think `rawSniffer` is not included with the python standard library.

Reply:
Yes indeed. But it is a class that was developed as a part of this module. Look at the file: netIDS/rawSniffer.py that is also part of this merge request.

Reply:
🙃True. Thanks.